test/test-visualizer.js

```diff
@@ -112,7 +112,9 @@ test('simple parse tree', function(t) {
     'change:grammarEditor': ['codeMirror'],
     'peek:failure': ['failure'],
     'unpeek:failure': [],
-    'goto:failure': ['failure']
+    'goto:failure': ['failure'],
+    'highlight:ruleDefinition': ['ruleName'],
```

Comment:
I'd like to name these `peek:ruleDefinition` and `unpeek:ruleDefinition`, and eventually add `goto:ruleDefinition`, just like we have with failures now.